src/pages/iou/IOUModal.js

```diff
@@ -82,6 +95,14 @@ class IOUModal extends Component {
         }));
     }
 
+    handleParticipants(userEmails) {
+        this.setState(prevState => ({
+            ...prevState,
```

Comment:
this can also be removed, we only need to pass `userEmails`

Reply:
@Julesssss  I have already removed it and as per Rory’s suggestion, renamed the function as well in the latest commit.

Reply:
Ah sorry, I missed that.